tools/ui-components/src/close-button/close-button.tsx

```diff
@@ -15,18 +15,41 @@ export function CloseButton({
   onClick
 }: CloseButtonProps): JSX.Element {
   const classes = [
-    'text-xl font-bold leading-none appearance-none opacity-20',
-    'hover:opacity-50 focus:opacity-50',
+    // Remove browser's default styles
+    'bg-transparent',
+    'border-none',
+    // Text styles
+    'text-lg',
```

Comment:
the 24px were too big for the setting buttons, and moving it to 23px was perfect to mimic the old styles.

Reply:
I will re-investigate it in the `Button` PR 🤔, I remember an issue existing. 

And you are correct, if there are a set of buttons that don't have similar styles in the `Button`, it's better to add new styles and new value there, instead of creating one for this specific niche case, should have thought about it more thoroughly.

I will revert it back, and see which button it was and add something like `text-name` there.